### ISSUE TITLE
Extension should also scan external XRef maps

### ISSUE THREAD

**Issue by tintoy** [tintoy/docfx-assistant#4]
Just `.zip` based ones to start with, but later we could resolve URLs, too.

**Comment by vicancy** [comment]
After build, DocFX generates a file called `xrefmap.yml`, do you think this can be leveraged?

**Comment by tintoy** [comment]
Yep, definitely! I think the project file already lists these (e.g. the ones for MSDN) so it's just a matter of unpacking and reading them...
